apps/web/app/deploy/[databaseId]/page.tsx

```diff
@@ -108,21 +107,14 @@ export default function Page() {
   }, [deploy])
 
   return (
-    <ParticlesBackground>
-      <Dialog open>
-        <DialogContent
-          className="max-w-3xl bg-background/70 backdrop-blur-sm"
-          showCloseButton={false}
-          overlay={false}
-        >
-          <DialogHeader>
-            <DialogTitle className="flex gap-2 items-center">
-              <SupabaseIcon />
-              Deploying your database
-            </DialogTitle>
-            <div className="py-2 border-b" />
-          </DialogHeader>
-          <div className="flex flex-col gap-6">
+    <div className="flex items-center justify-center h-full w-full bg-muted">
+      <div className="w-full h-full max-h-[400px] border border rounded-lg overflow-hidden relative flex justify-center items-center">
+        <div className="bg-background border rounded-md relative z-10 max-w-full">
+          <h1 className="flex gap-2 items-center border-b px-6 py-4 font-semibold">
+            <SupabaseIcon />
+            Deploying your database
+          </h1>
+          <div className="flex flex-col gap-6 p-6">
             <div className="flex items-center gap-4">
               <Loader2 className="animate-spin" />
```

Comment:
lets do it and see how it feels

Reply:
> > Another thing I noticed is that when I have databases created and I go back to the app, we're on the "home" page with the sidebar closed. I guess it's because of the new design but I was confused for 2 sec not seeing my previously created databases by default. What would help I think is showing the sidebar by default if there are existing databases on the home screen.

+1. On page load, if more than one database exists, sidebar open by default.

> On mobile, the database name is hidden in the header but I find it pretty important to identify which database I'm working with. I don't have a solution to provide as the header is already packed. 😅 I would probably push all the actions on the right to the "extra actions button" to gain some space and have the database name displayed instead. But that won't be trivial to implement for the "Deploy" button. 

I think this makes sense. Can be a follow up PR?

Reply:
Sidebar open by default if databases exist done in https://github.com/supabase-community/database-build/pull/146/commits/952c5a532c6f126d5685124c83dbf46e6d07f4dc

Reply:
> lets do it and see how it feels

Loader removed in https://github.com/supabase-community/database-build/pull/146/commits/3f478f262ce63d9b5708b7d787a6ec27d5108994